go-controller/pkg/ovn/egressip.go

```diff
@@ -1753,11 +1751,13 @@ func (oc *Controller) deleteNodeForEgress(node *v1.Node) error {
 // egress node experiences problems we want to move all egress IP assignment
 // away from that node elsewhere so that the pods using the egress IP can
 // continue to do so without any issues.
-func (oc *Controller) initClusterEgressPolicies(nodes []interface{}) {
+func (oc *Controller) initClusterEgressPolicies(nodes []interface{}) error {
 	v4ClusterSubnet, v6ClusterSubnet := getClusterSubnets()
 	oc.createDefaultNoReroutePodPolicies(v4ClusterSubnet, v6ClusterSubnet)
 	oc.createDefaultNoRerouteServicePolicies(v4ClusterSubnet, v6ClusterSubnet)
 	go oc.checkEgressNodesReachability()
+	// TODO(FF): make this function (1) idem-potent and (2) able to return upon failures
```

Comment:
so createDefaultNoReroutePodPolicies and createDefaultNoReroutServicePolicies should be idempotent right? they both just try to create or update an LRP in ovn. The only other thing in this is the goroutine that gets started (note that goroutine shoudl be using oc.stopChan to detect it needs to quit, but thats for another PR later) which cant fail. So I think you can just change the 2 create functions to return error and then return the error here as well?

Reply:
ack. yes. will do.